styles/scss/measureNumberFormat.scss

```diff
@@ -0,0 +1,186 @@
+// (C) 2020 GoodData Corporation
+@import "~@gooddata/goodstrap/lib/core/styles/variables.scss";
+@import "~@gooddata/goodstrap/lib/core/styles/globals.scss";
+
+.gd-measure-number-format-dropdown-body {
+    padding: 5px 0;
+
+    .gd-format-preset {
+        width: 200px;
+        padding: 2.5px 10px;
+
+        & > .gd-format-preset-name {
+            margin-left: 5px;
+        }
+
+        & > .gd-format-preset-preview {
+            color: $gd-color-text-dimmed;
+            font-weight: normal;
+            margin-right: 5px;
+            text-align: right;
+        }
+    }
+}
+
+$dialog_width: 250px;
```

Comment:
is it possible to these variables to pollute global scope and possibly conflict with the same name elsewhere? A more specific name could prevent it.

Reply:
...and the same for the one below